src/app/textbook/components/textbook-menu/textbook-menu.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, ViewChild } from '@angular/core';
+import { MatMenuTrigger } from '@angular/material/menu';
+import { DomSanitizer } from '@angular/platform-browser';
+import { MatIconRegistry } from '@angular/material/icon';
+import { settingsIcon } from './const';
+import { Store } from '@ngrx/store';
+import { AppState } from 'src/app/redux/models/state.model';
+import { setWordSettingsAddButtons, setWordSettingsTranslation } from 'src/app/redux/actions/textbooks.actions';
+import { selectIsAuthorized } from 'src/app/redux/selectors/user.selector';
+
+@Component({
+  selector: 'app-textbook-menu',
+  templateUrl: './textbook-menu.component.html',
+  styleUrls: ['./textbook-menu.component.scss'],
+})
+export class TextbookMenuComponent {
+  isCheckedTranslation = false;
+  isCheckedAddButtons = false;
+  labelPosition: 'before' | 'after' = 'after';
```

Comment:
It's better to create separate type for example ```type Position = 'before' | 'after'```

Reply:
Done.